.github/workflows/linux_cuda_wheel.yaml

```diff
@@ -0,0 +1,136 @@
+          path: pytorch/torchcodec/dist/
+      - name: Setup miniconda using test-infra
+        uses: ahmadsharif1/test-infra/.github/actions/setup-miniconda@14bc3c29f88d13b0237ab4ddf00aa409e45ade40
+        with:
+          python-version: ${{ matrix.python-version }}
+          default-packages: "conda-forge::ffmpeg=${{ matrix.ffmpeg-version-for-tests }}"
+      - name: Check env
+        run: |
+          ${CONDA_RUN} env
+          ${CONDA_RUN} conda info
+          ${CONDA_RUN} nvidia-smi
+      - name: Update pip
+        run: ${CONDA_RUN} python -m pip install --upgrade pip
+      - name: Install PyTorch
+        run: |
+          ${CONDA_RUN} python -m pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu${{ env.cuda_version_without_periods }}
+          ${CONDA_RUN} python -c 'import torch; print(f"{torch.__version__}"); print(f"{torch.__file__}"); print(f"{torch.cuda.is_available()=}")'
+      - name: Install torchcodec from the wheel
+        run: |
+          wheel_path=`find pytorch/torchcodec/dist -type f -name "*.whl"`
+          echo Installing $wheel_path
+          ${CONDA_RUN} python -m pip install $wheel_path -vvv
+
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Install cuda runtime dependencies
+        run: |
+          ${CONDA_RUN} conda install --yes nvidia::libnpp nvidia::cuda-nvrtc=12.4 nvidia::cuda-toolkit=12.4 nvidia::cuda-cudart=12.4 nvidia::cuda-driver-dev=12.4
+      - name: Install test dependencies
+        run: |
+          ${CONDA_RUN} python -m pip install --pre torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
+          # Ideally we would find a way to get those dependencies from pyproject.toml
+          ${CONDA_RUN} python -m pip install numpy pytest pillow
+
+      - name: Delete the src/ folder just for fun
+        run: |
+          # The only reason we checked-out the repo is to get access to the
+          # tests. We don't care about the rest. Out of precaution, we delete
+          # the src/ folder to be extra sure that we're running the code from
+          # the installed wheel rather than from the source.
+          # This is just to be extra cautious and very overkill because a)
+          # there's no way the `torchcodec` package from src/ can be found from
+          # the PythonPath: the main point of `src/` is precisely to protect
+          # against that and b) if we ever were to execute code from
+          # `src/torchcodec`, it would fail loudly because the built .so files
+          # aren't present there.
+          rm -r src/
+          ls
+      - name: Smoke test
+        run: |
+          ${CONDA_RUN} python test/decoders/manual_smoke_test.py
+      - name: Run Python tests
+        run: |
+          ${CONDA_RUN} FAIL_WITHOUT_CUDA=1 pytest test -vvv
+          ${CONDA_RUN} python benchmarks/decoders/gpu_benchmark.py --devices=cuda:0,cpu --resize_devices=none
```

Comment:
I couldn't figure it out from the logs: how long does it take to run the benchmarks? This CUDA job is fairly long, about 9mins, compared to just ~2 minutes for the CPU equivalent.

If the extra time is due to the additional dependencies installation steps then there isn't much we can do about it, but if it's caused by the benchmark, maybe we can make it optional / not run it on all PRs?

Reply:
The benchmarks are quite small compared to the rest of the install process. Torch install is probably the slowest step. I added a time command prefix so we know how much time that takes

Reply:
FYI, the benchmark takes around 7 seconds:

![image](https://github.com/user-attachments/assets/e25595c0-6df6-4448-9b9f-76f6339ac4b6)

The whole job is long because of mainly building ffmpeg and setting up environment, etc.